[FR] Create `osu!stream` (#6330)
ppy/osu-wiki: wiki/osu!stream/fr.md

| **PerfectHitValue** | (Number of misses + Number of 50s + Number of 100s + Number of 300s) \* 4 |

En d'autres termes, chaque miss vaut 0% de précision, chaque 50 vaut 25%, chaque 100 vaut 50%, et chaque 300 vaut 100%.

### Score

Le score donné par chaque note et la fin d'un slider est calculé avec la formule suivante :

`Score = Hit Value + Hit Value * (Combo multiplier * Fixed multiplier) / 25`

| Terme | Signification |
| :-: | :-- |
| **Hit Value** | Le hit circle judgement (50, 100 ou 300). |
| **Combo multiplier** | (Combo avant ce hit - 1) ou 0 ; le plus élevé des deux. |
| **Fixed multiplier** | Le multiplicateur qui fixe le score à 600.000 |

En outre, chaque tick de début, de fin et de répétition du slider rapporte 30 points et chaque tour d'un spinner rapporte 100 points.
Il y a un bonus supplémentaire de 10 points en fonction de la vitesse à laquelle vous faites tourner le spinner une fois que le spinner meter est plein.

### Hit circle judgement

**Hit-circles**

- Un cercle de 300, 100 ou 50 est attribué à partir d'une frappe normale, selon la précision de la frappe.
- Un miss est donné si un cercle a été frappé trop tôt ou n'a pas été frappé du tout.

>   - Si vous tapez dessus trop tôt, l'objet va trembler, on parle ici d'un [Notelock](https://osu.ppy.sh/wiki/fr/Gameplay/Notelock). Techniquement, il ne se passe rien, si ce n'est que l'on peut voir que l'objet a été touché trop tôt.

Reviewer: ```suggestion
  - Si vous tapez dessus trop tôt, l'objet va trembler, on parle ici d'un [Notelock](/wiki/Gameplay/Notelock). Techniquement, il ne se passe rien, si ce n'est que l'on peut voir que l'objet a été touché trop tôt.
```

- C = Une précision de plus de 60%
- D = Tout le reste

### Life meter

Contrairement à osu!, la barre de vie est différente dans osu!stream car elle est basée sur le mode de jeu.

En **Mode Easy, la barre de vie est complètement supprimée**.

![](img/Os-SUN.jpg "Exemple de Stream up! depuis une difficulté Normal.")

En **Stream Mode**, le joueur dispose de trois barres de vie et le joueur commence avec la deuxième barre de vie à moitié remplie au départ. Les joueurs se déplacent entre ces barres de vie grâce aux systèmes **Stream Up!** et **Stream Down!**, qui s'activent lorsqu'ils remplissent la barre de vie ou la vident complètement. Après une annonce, le jeu passe au nouveau "flux" en changeant l'arrière-plan et en passant de manière transparente à une difficulté supérieure ou inférieure de la beatmap, ainsi qu'en changeant de barre de vie. Lorsqu'un Stream Down! se produit, le joueur bénéficie d'une invincibilité temporaire pendant que la barre de vie passe à celle correspondant à la difficulté inférieure.

- Par défaut, **la barre de vie ne se vide pas tant que l'on joue**.

En **Mode Expert, une seule barre de vie complète est utilisée à la place**.

> Ce qui suit remplit la barre de vie :
> - Compléter les spinners et les Hold Bubbles.
> - Frapper un osu!, gold good!, green good!, 300, ou un 100, 
>   - Pour les cercles, les meilleurs jugements remplissent davantage la barre de vie. Le dernier cercle frappé d'une combinaison de couleurs donne plus de vie.
>   - Pour les sliders, la barre de vie est remplie en fonction du nombre de parts réalisées, une fois le slider terminé.

Reviewer: ```suggestion
Ce qui suit remplit la barre de vie : 

- Compléter les spinners et les Hold Bubbles.
- Frapper un osu!, gold good!, green good!, 300, ou un 100, 
  - Pour les cercles, les meilleurs jugements remplissent davantage la barre de vie. Le dernier cercle frappé d'une combinaison de couleurs donne plus de vie.
  - Pour les sliders, la barre de vie est remplie en fonction du nombre de parts réalisées, une fois le slider terminé.
```

# osu!stream

![](img/Os-Logo.jpg "logo d'osu!stream")

**osu!stream** est une version d'osu! créée de toutes pièces et spécialement conçue pour les appareils tactiles. Elle est actuellement disponible uniquement sur la plateforme [iOS](https://fr.wikipedia.org/wiki/IOS) d'Apple. Un portage [Android](https://fr.wikipedia.org/wiki/Android) était prévu et presque terminé, mais il n'a pas été publié en raison de problèmes de latence audio dans Android, qui rendaient le jeu peu agréable.

Il a introduit un tout nouveau système de jeu, avec de nouveaux éléments de jeu tels que des hold circles et des connected beats. Il ne prend pas en charge les beatmaps classiques d'osu! sans conversion manuelle. Les beatmaps ont été créées par une équipe spéciale de mappeurs afin de tirer parti des nouveaux éléments et de respecter un [niveau de normes plus strict](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).

> [osu!stream a été élaboré dans osu!monthly \#5 avec une interview.](https://osu.ppy.sh/community/forums/topics/59924)

Reviewer: ```suggestion
[osu!stream a été élaboré dans osu!monthly \#5 après une interview](https://osu.ppy.sh/community/forums/topics/59924)
```

Le développement de cette version s'est terminé en 2020. Pour plus d'informations, lisez le [billet de blog](https://blog.ppy.sh/osu-stream-2020-release/) sur sa dernière version. Le code source ainsi que les outils de conversion et de test des beatmaps ont été mis à disposition la même année sous une licence permissive sur [GitHub](https://github.com/ppy/osu-stream) pour les curieux, et toutes les beatmaps précédemment payantes sont désormais disponibles gratuitement dans la "boutique" du jeu.

## Installation

![](img/Os-Devices.jpg "osu!stream dans iDevice")

### Méthode de l'App Store de l'iDevice (recommandé)

**Assurez-vous que l'iDevice peut se connecter à Internet (soit par Wi-Fi, soit par 3G/4G).**

- Allez sur l'"App Store" depuis votre iDevice.
- Recherchez osu!stream
- Appuyez et installez osu!stream

>   - Si vous êtes invité à saisir votre identifiant Apple et votre mot de passe, faites le.

Reviewer 2: ```suggestion
  - Si vous êtes invité à saisir votre identifiant Apple et votre mot de passe, faites-le.
```

- osu!stream peut être joué une fois le téléchargement et l'auto-installation terminés.

>   - Il est suggéré de lire également la section "Ajouter des Beatmaps" pour obtenir le dernier pack de chansons gratuites. Le téléchargement actuel ne contient que 2 musiques.

Reviewer 2: ```suggestion
  - Il est suggéré de lire également la section "Ajouter des beatmaps" pour obtenir le dernier pack de musiques gratuites. Le téléchargement actuel ne contient que 2 musiques.
```

Reviewer 2: "Ajouter des beatmaps" est-il écrit en français ? Étant donné que les boutons mentionnés ci-dessous sont écrits en anglais, je doute

Reviewer: C'est la section de l'article que l'on évoque ici.

### Méthode USB

**L'appareil doit avoir [iTunes](https://www.apple.com/itunes/) installé.**

> - Allez sur [ce lien et téléchargez osu!stream](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)

Reviewer: ```suggestion
- Allez sur [ce lien](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8) et téléchargez osu!stream
```

  - Si l'on vous demande votre identifiant Apple, saisissez votre identifiant Apple et votre mot de passe.
- Branchez l'iDevice sur l'appareil (qui a osu!stream).
- Transférez les fichiers osu!stream en utilisant iTunes.
- Débranchez l'iDevice et osu!stream pourra être lancé.

> ## Ajouter des Beatmaps

Reviewer 2: ```suggestion
## Ajouter des beatmaps
```

**Assurez-vous que l'iDevice peut se connecter à Internet (soit par Wi-Fi, soit par 3G/4G).**

- Ouvrez osu!stream.

> - Appuyez sur le symbole d'osu! et vous devriez être redirigé vers le menu principal (ou le tutoriel si c'est la première fois)

Reviewer 2: ```suggestion
- Appuyez sur le symbole d'osu! et vous devriez être redirigé vers le menu principal (ou le tutoriel si c'est la première fois).
```

- Appuyez sur la barre "Store".

>   - Sinon, allez dans "Play" et appuyez sur "Download more songs!"

Reviewer 2: ```suggestion
  - Sinon, allez dans "Play" et appuyez sur "Download more songs!".
```

- Sélectionnez un pack souhaité.

>   - Il est recommandé de télécharger d'abord les packs qui ne coûtent pas d'argent. Les packs gratuits ont l'étiquette "FREE" au lieu de l'étiquette de coût "USD$".

Reviewer 2: Je vois pas cette info sur l'article anglais, elle a été rajoutée ?

>   - Il est recommandé de télécharger d'abord les packs qui ne coûtent pas d'argent. Les packs gratuits ont l'étiquette "FREE" au lieu de l'étiquette de coût "USD$".

Reviewer: ```suggestion
```

C'est quand j'ai du faire la remise en conformité par rapport à #6360, j'ai du oublier de supprimer cette ligne.

## Quelle est la différence ?

- **Jouable sur tous les appareils iOS uniquement** (pour l'instant).
- Les beatmaps sont créées de toutes pièces.
- Aucun compte n'est nécessaire pour télécharger les packs de musiques.
- Possibilité d'intégrer osu!stream avec un compte Twitter.
- Le meilleur score peut être envoyé anonymement sur le site sous le nom de "Guest" (nécessite une connexion Internet).

> - Le timer circulaire est remplacé et déplacé vers le bas avec une petite ligne de bordure qui est colorée en fonction de la précision du jeu. \[Bottom(Long)\]

Reviewer 2: ```suggestion
- Le timer circulaire est remplacé et déplacé vers le bas avec une petite ligne de bordure qui est colorée en fonction de la précision de la partie. \[Bottom(Long)\]
```

> - **Le timing moyen** est indiqué sur la page des notes.

Reviewer 2: ```suggestion
- **Le timing moyen** est indiqué sur l'écran de résultat.
```

> - Les anciens enregistrements et replays ne peuvent pas être sauvegardés.

Reviewer 2: ```suggestion
- Les anciens scores et replays ne peuvent pas être sauvegardés.
```

- On ne peut pas changer de skins ni appliquer de mods (pour l'instant).
- Des Easter Eggs peuvent être trouvés mais il n'y a pas de page de succès...

### Nouveau système de gameplay

**Note :** Testez les nouveaux éléments de gameplay dans l'onglet "Tutorial".

#### Affichage du Finger Guide

![](img/Os-FG.jpg "Affichage du Finger Guides")

> - Les deux cercles transparents qui apparaissent dans le jeu ont été créés pour aider les débutants à savoir quand frapper les objets.

Reviewer 2: ```suggestion
- Les deux cercles transparents qui apparaissent en jeu ont été créés pour aider les débutants à savoir quand toucher les objets.
```

>   - Il indique la méthode suggérée pour aborder la conception de la beatmap avec perfection.

Reviewer 2: ```suggestion
  - Cela indique la méthode suggérée pour aborder la conception de la beatmap avec perfection.
```

- La couleur verte est pour la main gauche et la couleur rouge est pour la main droite.
- Cette fonction peut être désactivée dans le menu des options ou après avoir terminé le tutoriel.

#### Connected Beats

![](img/Os-CB.png "Connected Beats")

- Les Connected Beats, comme son titre l'indique, exige du joueur qu'il appuie sur les deux objets en même temps.
  - Les Connected Beats ne sont pas limités à juste "hit circle-hit circle", ils peuvent aussi être "hit circle-slider", "hold circle-slider", etc.
- Les Connected Beats peuvent être identifiés par une ligne blanche droite reliée à un autre objet.

#### Hold circle

![](img/Os-HC.jpg "Hold circle 1")

---

![](img/Os-HC2.jpg "Hold circle 2")

> - Les Hold circle demandent au joueur de maintenir le cercle jusqu'à ce qu'il "explose".

Reviewer 2: ```suggestion
- Les Hold circles demandent au joueur de maintenir le cercle jusqu'à ce qu'il "explose".
```

> - La précision est déterminée par le début ou la fin du clic et le temps de maintien.

Reviewer 2: ```suggestion
- La précision est déterminée selon le timing du joueur (touché tôt ou tard). Elle dépendra également de si le joueur l'a tenu jusqu'au bout ou non.
```

Reviewer 2: La deuxième phrase a peut-être besoin de rewording

  - Si le cercle a été relâché avant qu'il n'explose, le multiplicateur de combo sera remis à 0, mais le Hold circle n'entraînera pas un miss à partir du moment où le joueur l'a touché.
  - Si le joueur n'a pas tenu le cercle, les HP seront pris tant que le cercle n'a pas explosé.

### De nouveaux modes pour remplacer la difficulté  

---

![](img/Os-PlayEasy.png "Easy")

---

![](img/Os-PlayNormal.png "Normal")

---

![](img/Os-PlayHard.png "Hard")

---

![](img/Os-PlayExpert.jpg "Expert")

---

> #### Easy

Reviewer 2: ```suggestion
#### Mode Easy
```

![](img/Os-ModeEasy.jpg "Easy mode")

**Difficulté : Easy + [NoFail](/wiki/Game_modifier/No_Fail)**

> Le mode Easy permet aux nouveaux joueur de s'habituer au gameplay d'osu!stream. Le jeu est considérablement plus lent pour que les nouveaux joueurs comprennent le déroulement et le timing des frappes. Il n'y a pas de barre de vie et le mode Easy ne débloque pas le mode Expert.

Reviewer 2: ```suggestion
Le mode Easy permet aux nouveaux joueur de s'habituer au gameplay d'osu!stream. Le jeu est considérablement plus lent pour que les nouveaux joueurs comprennent le déroulement et le timing. Il n'y a pas de barre de vie et le mode Easy ne débloque pas le mode Expert.
```

La couleur de fond est le vert.

#### Stream mode

![](img/Os-ModeStream.jpg "Stream mode")

**Difficulté : Easy ("Stream Down!" à partir de Normal) ~ Normal (point de départ) ~ Hard ("Stream Up!" à partir de Normal)**

> Le mode Stream est un mode exclusif à osu!stream. Dans ce mode, les joueurs commencent avec une barre de vie "2nd" à moitié remplie et un réglage de difficulté normal. Le joueur doit ensuite la remplir complètement pour déclencher le mode "Stream Up!", qui augmente la difficulté (en ajoutant des objets et en passant à la difficulté Hard de la beatmap), mais qui reste nettement moins difficile que le mode Expert. Lorsque les joueurs entrent dans le mode "Stream Up!", une nouvelle barre de vie est émise et "Stream Down!" se déclenche si cette barre de vie est épuisée, ce qui ramène à la difficulté Normal ou Easy et à la barre de vie précédente.

Reviewer 2: ```suggestion
Le mode Stream est un mode exclusif à osu!stream. Dans ce mode, les joueurs commencent avec une "deuxième" barre de vie à moitié remplie et un réglage de difficulté normal. Le joueur doit ensuite la remplir complètement pour déclencher le mode "Stream Up!", qui augmente la difficulté (en ajoutant des objets et en passant à la difficulté Hard de la beatmap), mais qui reste nettement moins difficile que le mode Expert. Lorsque les joueurs entrent dans le mode "Stream Up!", une nouvelle barre de vie est émise et "Stream Down!" se déclenche si cette barre de vie est épuisée, ce qui ramène à la difficulté Normal ou Easy et à la barre de vie précédente.
```

En bref, le joueur commence entre Easy et Hard.

> Notez que le joueur **doit obtenir au moins le rang A (>80% de précision) dans ce mode afin de débloquer le mode Expert de la chanson**. Une précision inférieure à 80 %, même en full-combo, ne compte pas comme A (elle sera comptée comme B, comme d'habitude).

Reviewer 2: ```suggestion
Notez que le joueur **doit obtenir au moins le rang A (>80% de précision) dans ce mode afin de débloquer le mode Expert de la musique**. Une précision inférieure à 80 %, même en full-combo, ne compte pas comme un rang A (elle sera comptée comme un rang B, comme d'habitude).
```

## Interface

---

![](img/Os-MM.png "Menu principal")

---

![](img/Os-SL.jpg "Liste des musiques")

---

![](img/Os-Store.jpg "Écran du magasin")

---

![](img/Os-DS.png "Sélection du menu de style")

---

![](img/Os-SI.png "Informations sur les musiques")

---

![](img/Os-PlayExpert.jpg "Expert")

---

![](img/Os-Pause.png "Menu de pause")

---

![](img/Os-FM.png "Menu d'échec")

---

![](img/Os-SC.png "Stage clean !")

---

![](img/Os-Grade.jpg "Écran des grades")

---

![](img/Os-O1.png "Écran des options 1")

---

![](img/Os-O2.png "Écran des options 2")

---

## Liste des Chansons

***[Cliquez ici pour une liste complète](Song_List)***

## Score

*Attention : Le classement peut toujours être consulté en jeu, mais aucun nouveau score ne sera accepté.*

### Précision

`Accuracy = HitValue / (PerfectHitValue)`

| Terme | Formule |
| :-: | :-- |
| **HitValue** | Number of misses \* 0 + Number of 50s \* 1 + Number of 100s * 2 + Number of 300s \* 4) |
| **PerfectHitValue** | (Number of misses + Number of 50s + Number of 100s + Number of 300s) \* 4 |

En d'autres termes, chaque miss vaut 0% de précision, chaque 50 vaut 25%, chaque 100 vaut 50%, et chaque 300 vaut 100%.

### Score

> Le score donné par chaque note et la fin d'un slider est calculé avec la formule suivante :

Reviewer 2: ```suggestion
Le score donné par chaque cercle et la fin d'un slider est calculé avec la formule suivante :
```

`Score = Hit Value + Hit Value * (Combo multiplier * Fixed multiplier) / 25`

| Terme | Signification |
| :-: | :-- |
| **Hit Value** | Le hit circle judgement (50, 100 ou 300). |
| **Combo multiplier** | (Combo avant ce hit - 1) ou 0 ; le plus élevé des deux. |
| **Fixed multiplier** | Le multiplicateur qui fixe le score à 600.000 |

> En outre, chaque tick de début, de fin et de répétition du slider rapporte 30 points et chaque tour d'un spinner rapporte 100 points.

Reviewer 2: ```suggestion
En outre, chaque début, fin et répétition de slider rapporte 30 points et chaque tour d'un spinner rapporte 100 points.
```

Il y a un bonus supplémentaire de 10 points en fonction de la vitesse à laquelle vous faites tourner le spinner une fois que le spinner meter est plein.

### Hit circle judgement

**Hit-circles**

> - Un cercle de 300, 100 ou 50 est attribué à partir d'une frappe normale, selon la précision de la frappe.

Reviewer 2: ```suggestion
- Un score de 300, 100 ou 50 sera attribué selon la précision avec laquelle le joueur appuyé sur un cercle normal.
```

> - Un miss est donné si un cercle a été frappé trop tôt ou n'a pas été frappé du tout.

Reviewer 2: ```suggestion
- Un miss est donné si un cercle a été touché trop tôt ou n'a pas été touché du tout.
```

>   - Si vous tapez dessus trop tôt, l'objet va trembler, on parle ici d'un [Notelock](/wiki/Gameplay/Notelock). Techniquement, il ne se passe rien, si ce n'est que l'on peut voir que l'objet a été touché trop tôt.

Reviewer 2: ```suggestion
  - Si vous le touchez trop tôt, l'objet va trembler, on parle ici d'un [Notelock](/wiki/Gameplay/Notelock). Techniquement, il ne se passe rien, si ce n'est que l'on peut voir que l'objet a été touché trop tôt.
```

**Hold-circle**

> - Le jugement du hold circle dépend de la frappe initial et du temps de maintien avant l'explosion.

Reviewer 2: ```suggestion
- Le jugement du hold circle dépend de la précision avec laquelle il a été touché et du temps de maintien avant l'explosion.
```

>   - 300 si c'est fait parfaitement.

Reviewer 2: ```suggestion
  - 300 si réalisé parfaitement.
```

>   - 100 pour une prise initiale moins précise et une tenue correcte.

Reviewer 2: ```suggestion
  - 100 pour avoir touché moins précisément et un maintien correct.
```

>   - 50 pour au moins le toucher.

Reviewer 2: ```suggestion
  - 50 pour au moins l'avoir touché
```

  - 30 pour chaque explosion.

>   - Le multiplicateur est remis à zéro si les cercles n'ont pas été tenus correctement et un miss pour ne pas être tenus du tout.

Reviewer 2: ```suggestion
  - Le multiplicateur est remis à zéro si les cercles n'ont pas été tenus correctement et donnent un miss s'ils n'ont pas été tenus du tout.
```

**Sliders**

> - **Le jugement des sliders dépend de la précision de la frappe initial**.

Reviewer 2: ```suggestion
- **Le jugement des sliders dépend de la précision avec laquelle le slider a été touché**.
```

- Les sliders sont constitués de ticks de sliders, qui comprennent les points de début, de fin et de répétition du slider.

>   - Un 300 pour avoir réussi à obtenir tous les ticks du slider et une frappe initiale précise.

Reviewer 2: ```suggestion
  - Un 300 pour avoir réussi à obtenir tous les ticks du slider et pour avoir touché le début du slider avec précision.
```

>   - Un 100 si au moins la moitié des sliders parts ont été frappés par une frappe initiale assez précise.

Reviewer 2: ```suggestion
  - Un 100 si au moins la moitié des parties du slider ont été obtenues tout en ayant touché le début du slider assez précisément.
```

>   - Un 50 pour avoir réussi à toucher ne serait-ce qu'un seul des ticks ou la frappe initiale.

Reviewer 2: ```suggestion
  - Un 50 pour avoir réussi à toucher ne serait-ce qu'un seul des ticks ou le début du slider.
```

> - **Cliquer sur un slider trop tôt** n'est pas considéré comme un miss, mais **réinitialise le multiplicateur de score à 0**.

Reviewer 2: ```suggestion
- **Toucher un slider trop tôt** n'est pas considéré comme un miss, mais **réinitialise le multiplicateur de score à 0**.
```

> - Un miss si vous n'avez pas fait le slider.

Reviewer 2: ```suggestion
- Un miss si vous n'avez pas touché le slider du tout.
```

**Spinner**

- Pour un spinner, un 300, 100 ou 50 selon le nombre de tours effectués par rapport à la longueur du spinner.
- Un miss si l'on ne parvient pas à remplir le compteur indiqué afin de valider le spinner.

### Multiplicateur

Les éléments suivants ajoutent chacun un point au multiplicateur de score :

> - Des frappes sur un cercle.

Reviewer 2: ```suggestion
- Toucher un cercle.
```

- Le début, la fin et chaque hold tick d'un hold circle.
- Le début, la fin et chaque tick d'un slider.
- La réalisation d'un spinner.

Les éléments suivants remettent le multiplicateur de score à zéro :

> - Manquer un hit circle.

Reviewer 2: ```suggestion
- Manquer un cercle.
```

- Lâcher le hold circle alors qu'il n'a pas encore explosé.

> - Impossibilité de toucher simultanément les connected beats. (En toucher l'un, et pas l'autre)

Reviewer 2: ```suggestion
- Ne pas réussir à toucher simultanément les connected beats. (En toucher l'un, et pas l'autre)
```

- Cliquer sur le début d'un slider trop tôt.

> - Manquer le début d'un slider ou le tick d'un slider.

Reviewer 2: ```suggestion
- Manquer le début d'un slider ou un tick d'un slider.
```

> - Impossible de remplir la jauge d'un spinner avec le statut clear.

Reviewer 2: ```suggestion
- Ne pas réussir à remplir la jauge d'un spinner avec le statut clear.
```

Ce qui suit n'augmente pas ou ne réinitialise pas le multiplicateur de score :

- Manquer la fin d'un slider. (Vous obtiendrez "Good!" ou 100/50 Hit).
- Le bonus de score du spinner

### Grades

![](img/Os-Grade.jpg "Écran des grades")

**Grades normaux**

- SS = 100% de précision
- S = Une précision de plus de 90 %
- A = Une précision de plus de 80%
- B = Une précision de plus de 70%
- C = Une précision de plus de 60%
- D = Tout le reste

> ### Life meter

Reviewer 2: ```suggestion
### Barre de vie
```

Contrairement à osu!, la barre de vie est différente dans osu!stream car elle est basée sur le mode de jeu.

En **Mode Easy, la barre de vie est complètement supprimée**.

![](img/Os-SUN.jpg "Exemple de Stream up! depuis une difficulté Normal.")

> En **Stream Mode**, le joueur dispose de trois barres de vie et le joueur commence avec la deuxième barre de vie à moitié remplie au départ. Les joueurs se déplacent entre ces barres de vie grâce aux systèmes **Stream Up!** et **Stream Down!**, qui s'activent lorsqu'ils remplissent la barre de vie ou la vident complètement. Après une annonce, le jeu passe au nouveau "flux" en changeant l'arrière-plan et en passant de manière transparente à une difficulté supérieure ou inférieure de la beatmap, ainsi qu'en changeant de barre de vie. Lorsqu'un Stream Down! se produit, le joueur bénéficie d'une invincibilité temporaire pendant que la barre de vie passe à celle correspondant à la difficulté inférieure.

Reviewer 2: ```suggestion
En **Stream Mode**, le joueur dispose de trois barres de vie et le joueur commence avec la deuxième barre de vie à moitié remplie au départ. Les joueurs se déplacent entre ces barres de vie grâce aux systèmes **Stream Up!** et **Stream Down!**, qui s'activent lorsqu'ils remplissent la barre de vie ou la vident complètement. Après une annonce, le jeu passe au nouveau "stream" en changeant l'arrière-plan et en passant de manière transparente à une difficulté supérieure ou inférieure de la beatmap, ainsi qu'en changeant de barre de vie. Lorsqu'un Stream Down! se produit, le joueur bénéficie d'une invincibilité temporaire pendant que la barre de vie passe à celle correspondant à la difficulté inférieure.
```

- Par défaut, **la barre de vie ne se vide pas tant que l'on joue**.

En **Mode Expert, une seule barre de vie complète est utilisée à la place**.

Ce qui suit remplit la barre de vie : 

- Compléter les spinners et les Hold Bubbles.

> - Frapper un osu!, gold good!, green good!, 300, ou un 100, 

Reviewer 2: ```suggestion
- Obtenir un osu!, gold good!, green good!, 300, ou un 100, 
```

>   - Pour les cercles, les meilleurs jugements remplissent davantage la barre de vie. Le dernier cercle frappé d'une combinaison de couleurs donne plus de vie.

Reviewer 2: ```suggestion
  - Pour les cercles, les meilleurs jugements remplissent davantage la barre de vie. Le dernier cercle touché d'une combinaison de couleurs donne plus de vie.
```

>   - Pour les sliders, la barre de vie est remplie en fonction du nombre de parts réalisées, une fois le slider terminé.

Reviewer 2: ```suggestion
  - Pour les sliders, la barre de vie est remplie en fonction du nombre de parties touchées, une fois le slider terminé.
```

<!-- TODO: some updates and needs a proper end of life message. it's also open source now -->

# osu!stream

![](img/Os-Logo.jpg "osu!stream logo")

**osu!stream** est une version d'osu! créée de toutes pièces et spécialement conçue pour les appareils tactiles. Elle est actuellement disponible uniquement sur la plateforme [iOS](http://en.wikipedia.org/wiki/IOS) d'Apple, mais un portage [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) est prévu. Elle introduit un tout nouveau système de jeu, comme le hold circle, et les rythmes connectés.

> osu!stream ne prend pas en charge les beatmaps classiques d'osu!. Au lieu de cela, les beatmaps sont créées par une équipe spéciale de mappeurs pour tirer parti des nouveaux éléments et respecter un niveau de normes plus strict. Des beatmaps gratuites et payantes sont disponibles dans la "Boutique".

Reviewer 3: un niveau de normes plus strictes.

**Assurez-vous que l'iDevice peut se connecter à Internet (soit par Wi-Fi, soit par 3G/4G).**

- Allez sur "App Store" depuis votre iDevice.
- Recherchez osu!stream
- Appuyez et installez osu!stream
  - Si vous êtes invité à saisir votre identifiant Apple et votre mot de passe, veuillez le faire.
- osu!stream peut être joué une fois le téléchargement et l'auto-installation terminés.
  - Il est suggéré de lire également la section "Ajouter des Beatmaps" pour obtenir le dernier pack de chansons gratuites. Le téléchargement actuel ne contient que 2 chansons.

### Méthode USB

**L'appareil doit avoir [iTunes](https://www.apple.com/itunes/) installé.**

- Allez sur [ce lien et téléchargez osu!stream](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)
  - Si l'on vous demande votre identifiant Apple, saisissez votre identifiant Apple et votre mot de passe.
- Branchez l'iDevice sur l'appareil (qui a osu!stream).
- Transférez les fichiers osu!stream en utilisant iTunes.
- Débranchez l'iDevice et osu!stream peut être lancé.

## Ajouter des Beatmaps

**Assurez-vous que l'iDevice peut se connecter à Internet (soit par Wi-Fi, soit par 3G/4G).**

- Ouvrez osu!stream.

> - Appuyez sur le symbole osu! et vous devriez être dirigé vers le menu principal (ou le tutoriel si c'est la première fois)

Reviewer 3: not sure about this one
>et vous devriez être redirigé vers le menu principal

> - Appuyez sur le symbole osu! et vous devriez être dirigé vers le menu principal (ou le tutoriel si c'est la première fois)

Reviewer 3: ```suggestion
- Appuyez sur le symbole osu! et vous devriez être redirigé vers le menu principal (ou le tutoriel si c'est la première fois)
```

En bref, le joueur commence entre Easy et Hard.

Notez que le joueur **doit obtenir au moins le rang A (>80% de précision) dans ce mode afin de débloquer le mode Expert de la chanson**. Une précision inférieure à 80 %, même en full-combo, ne compte pas comme A (elle sera comptée comme B, comme d'habitude).

La couleur de fond est verte (Easy), bleue (Normal) et rose (Hard).

---

![](img/Os-SU.png "Stream Up!")

---

![](img/Os-SD.png "Stream Down!")

---

![](img/Os-Zero1.png "Critical health")

---

![](img/Os-Zero2.png "Game Over")

---

#### Mode Expert

![](img/Os-ModeExpert.jpg "Expert mode")

**Difficulté: Expert**

> Verrouillé par défaut et ne pouvant être déverrouillé qu'en terminant le mode Stream de la chanson avec un A ou mieux, le mode Expert est destiné aux joueurs qui veulent un plus grand défi. Dans ce mode, le joueur dispose d'une barre de vie pleine et doit la conserver jusqu'à la fin de la chanson. Le fait de le terminer ne débloquera rien, si ce n'est le droit de se vanter.

Reviewer 3: Verrouillée par défaut et ne pouvant être déverrouillée

Author: @Biokax tu peux directement request les changements au lieu de les suggérer stp :v ?

Reviewer 3: ```suggestion
Verrouillée par défaut et ne pouvant être déverrouillée qu'en terminant le mode Stream de la chanson avec un A ou mieux, le mode Expert est destiné aux joueurs qui veulent un plus grand défi. Dans ce mode, le joueur dispose d'une barre de vie pleine et doit la conserver jusqu'à la fin de la chanson. Le fait de le terminer ne débloquera rien, si ce n'est le droit de se vanter.
```

- osu!stream peut être joué une fois le téléchargement et l'auto-installation terminés.
  - Il est suggéré de lire également la section "Ajouter des Beatmaps" pour obtenir le dernier pack de chansons gratuites. Le téléchargement actuel ne contient que 2 chansons.

### Méthode USB

**L'appareil doit avoir [iTunes](https://www.apple.com/itunes/) installé.**

- Allez sur [ce lien et téléchargez osu!stream](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)
  - Si l'on vous demande votre identifiant Apple, saisissez votre identifiant Apple et votre mot de passe.
- Branchez l'iDevice sur l'appareil (qui a osu!stream).
- Transférez les fichiers osu!stream en utilisant iTunes.
- Débranchez l'iDevice et osu!stream peut être lancé.

## Ajouter des Beatmaps

**Assurez-vous que l'iDevice peut se connecter à Internet (soit par Wi-Fi, soit par 3G/4G).**

- Ouvrez osu!stream.
- Appuyez sur le symbole osu! et vous devriez être redirigé vers le menu principal (ou le tutoriel si c'est la première fois)
- Appuyez sur la barre "Store".
  - Sinon, allez dans "Play" et appuyez sur "Télécharger plus de chansons !"
- Sélectionnez un pack souhaité.

>   - Il est conseillé de télécharger d'abord les packs qui ne coûtent pas d'argent. Les packs gratuits ont l'étiquette "FREE" au lieu de l'étiquette de coût "USD$".

Reviewer 3: ```suggestion
  - Il est recommandé de télécharger d'abord les packs qui ne coûtent pas d'argent. Les packs gratuits ont l'étiquette "FREE" au lieu de l'étiquette de coût "USD$".
```

#### Affichage du Finger Guide

![](img/Os-FG.jpg "Finger Guides display")

- Les deux cercles transparents qui apparaissent dans le jeu ont été créés pour aider les débutants à savoir quand frapper les objets.
  - Il indique la méthode suggérée pour aborder la conception de la carte avec perfection.
- La couleur verte est pour la main gauche et la couleur rouge est pour la main droite.
- Cette fonction peut être désactivée dans le menu des options ou après avoir terminé le tutoriel.

#### Beats Connectés

![](img/Os-CB.png "Connected Beats")

- Les Beats Connectés, comme son titre l'indique, exige du joueur qu'il appuie sur les deux objets en même temps.
  - Les Beats Connectés ne sont pas limités à juste "hit circle-hit circle", ils peuvent aussi être "hit circle-slider", "hold circle-slider" etc.
- Les Beats Connectés peuvent être identifiés par une ligne blanche droite reliée à un autre objet.

#### Hold circle

![](img/Os-HC.jpg "Hold circle 1")

---

![](img/Os-HC2.jpg "Hold circle 2")

- Les Hold circle demandent au joueur de maintenir le cercle jusqu'à ce qu'il "explose".
- La précision est déterminée par le début ou la fin du clic et le temps de maintien.

>   - Si le cercle a été relâché avant qu'il n'explose, le multiplicateur de combo sera remis à 0, mais le Hold circle n'entraînera pas un échec tant que le joueur aura celui-ci.

Reviewer 3: ```suggestion
  - Si le cercle a été relâché avant qu'il n'explose, le multiplicateur de combo sera remis à 0, mais le Hold circle n'entraînera pas un miss à partir du moment où le joueur l'a touché.
```

| **Valeur parfaite** | (Nombre d'erreurs + Nombre de 50's + Nombre de 100's + Nombre de 300's) \* 4 |

En d'autres termes, chaque note manquée vers 0%, chaque 50 vers 25%, chaque 100 vers 50% et chaque 300 vers 100%.

### Score

Le score donné par chaque note et la fin d'un slider est calculé avec la formule suivante :

`Score = Hit Value + Hit Value * (Multiplicateur de combo * Multiplicateur fixe) / 25`

| Terme | Signification |
| :-: | :-- |
| **Valeur de la note** | Le jugement du cercle (50, 100 ou 300). |
| **Multiplicateur de combo** | (Combo avant ce hit - 1) ou 0 ; le plus élevé des deux. |
| **Multiplicateur fixe** | Le multiplicateur qui fixe le score à 600.000 |

En outre chaque début, fin and et tick d'un slider donne 30 points et chaque spin de spinner donne 100 points.
Un bonus additionnel de 10 points est donné dépendamment de la vitesse du spinner après que le spinner soit entièrement complété.

### Jugement des cercles

**Hit-circles**

- Un 300, 100 ou 50 d'un Hit-circle dépendant de la précision.
- Ne pas taper le hit-circle du tout, ou le taper trop tôt entrainera un miss.

>   - Si vous tapez dessus très, très tôt, l'objet va trembler. Techniquement, il ne se passe rien, si ce n'est que l'on peut voir que l'objet a été touché trop tôt.

Reviewer 3: ```suggestion
  - Si vous tapez dessus trop tôt, l'objet va trembler, on parle ici d'un [Notelock](https://osu.ppy.sh/wiki/fr/Gameplay/Notelock). Techniquement, il ne se passe rien, si ce n'est que l'on peut voir que l'objet a été touché trop tôt.
```

**Hold-circle**

- Le jugement du hold-circle dépend du clic initial et du temps de maintien avant l'explosion.
  - 300 si c'est fait parfaitement.
  - 100 pour une prise initiale moins précise et une tenue correcte.
  - 50 pour au moins, le toucher.
  - 30 pour chaque explosion.
  - Le multiplicateur est remis à zéro si l'on n'a pas tenu les cercles correctement et obtient un miss pour ne pas l'avoir fait.

**Sliders**

> - **Le jugement des sliders dépend de la précision du tapotement initial**.

Reviewer: ```suggestion
- **Le jugement des sliders dépend de la précision de la frappe initial**.
```

- Les sliders sont constitués de ticks de sliders, qui comprennent les points de début, de fin et de répétition du slider.

>   - Un 300 si vous réussissez à obtenir tous les ticks du slider et un tapotement initial précis.

Reviewer: ```suggestion
  - Un 300 pour avoir réussi à obtenir tous les ticks du slider et une frappe initiale précise.
```

>   - Un 100 si au moins la moitié d'entre eux avec un tapotement initial considérable.

Reviewer: ```suggestion
  - Un 100 si au moins la moitié des sliders parts ont été frappés par une frappe initiale assez précise.
```

>   - 50 si vous n'avez pas réussi à atteindre ne serait-ce qu'un seul des ticks ou le tapotement initial.

Reviewer: ```suggestion
  - Un 50 pour avoir réussi à toucher ne serait-ce qu'un seul des ticks ou la frappe initiale.
```

- **Cliquer sur un slider trop tôt** n'est pas considéré comme un miss, mais **réinitialise le multiplicateur de score à 0**.

> - Un miss si vous n'avez pas fait le curseur.

Reviewer: ```suggestion
- Un miss si vous n'avez pas fait le slider.
```

**Spinner**

- Pour un spinner, un 300, 100 ou 50 selon le nombre de tours effectués par rapport à la longueur du spinner.

> - Un échec si l'on ne parvient pas à remplir le compteur indiqué de part et d'autre du spinner.

Reviewer 3: ```suggestion
- Un miss si l'on ne parvient pas à remplir le compteur indiqué afin de valider le spinner.
```

### Multiplicateur

Les éléments suivants ajoutent chacun un point au multiplicateur de score :

> - Une touche réussie sur le hit circle.
> - Une prise réussie sur le hold circle.
> - Une touche réussie au début d'un slider.
> - Un touché réussi à la fin d'un slider.
> - Un touché réussi sur le tic-tac du slider.
> - Un spinner terminé avec succès.

Reviewer: ```suggestion
- Des frappes sur un cercle.
- Le début, la fin et chaque hold tick d'un hold circle.
- Le début, la fin et chaque tick d'un slider.
- La réalisation d'un spinner.
```

Les éléments suivants remettent le multiplicateur de score à zéro :

- Manquer un hit circle.
- Lâcher le hold circle alors qu'il n'a pas encore explosé.

> - Impossibilité de toucher simultanément les beats connectés. (En toucher un, et pas l'autre)

Reviewer: ```suggestion
- Impossibilité de toucher simultanément les connected beats. (En toucher l'un, et pas l'autre)
```

- Cliquer sur le début d'un slider trop tôt.

> - Manquer le début d'un slider ou le tic-tac d'un slider.

Reviewer: ```suggestion
- Manquer le début d'un slider ou le tick d'un slider.
```

Ça je sais que c'est du Google Traduction, car ça m'a déjà sorti ça quand je cherchais une traduction pour ce terme.

> - Impossible de remplir la jauge d'un spinner avec un statut clair.

Reviewer: ```suggestion
- Impossible de remplir la jauge d'un spinner avec le statut clear.
```

Ce qui suit n'augmente pas ou ne réinitialise pas le multiplicateur de score :

- Manquer la fin d'un slider. (Vous obtiendrez "Good!" ou 100/50 Hit).
- Le bonus de score du spinner

### Grades

> ![](img/Os-Grade.jpg "Ecran des grades")

Reviewer: ```suggestion
![](img/Os-Grade.jpg "Écran des grades")
```

**Grades normaux**

- SS = 100% de précision

> - S = Over 90% de précision

Reviewer: ```suggestion
- S = Une précision de plus de 90 %
```

> - A = Over 80% de précision

Reviewer: ```suggestion
- A = Une précision de plus de 80%
```

> - B = Over 70% de précision

Reviewer: ```suggestion
- B = Une précision de plus de 70%
```

> - C = Over 60% de précision

Reviewer: ```suggestion
- C = Une précision de plus de 60%
```

> - D = Le reste.

Reviewer: ```suggestion
- D = Tout le reste
```

### Life meter

> Contrairement à osu !, la barre de vie est différente dans osu!stream car elle est basée sur le mode de jeu.

Reviewer: ```suggestion
Contrairement à osu!, la barre de vie est différente dans osu!stream car elle est basée sur le mode de jeu.
```

> En **Mode facile, la barre de vie est complètement supprimée**.

Reviewer: ```suggestion
En **Mode Easy, la barre de vie est complètement supprimée**.
```

> ![](img/Os-SUN.jpg "Exemple de Stream up! depuis une difficulté normale.")

Reviewer: ```suggestion
![](img/Os-SUN.jpg "Exemple de Stream up! depuis une difficulté Normal.")
```

> En **Mode Stream** cependant, le joueur reçoit **trois barres de vie et le joueur commence avec la deuxième barre de vie à moitié remplie au départ**. Le joueur (gagne/perd) la barre de vie par un message (Stream Up!/Stream Down !) pour (remplir la barre de vie complètement/utiliser tous les HP de cette barre de vie). Après l'affichage du message, le jeu affiche une animation en arrière-plan qui augmente ou diminue la difficulté et affiche une barre de vie à moitié remplie. Si un joueur reçoit un "Stream Down !", il sera temporairement invisible (en raison de l'absence de santé dans la barre de vie utilisée) jusqu'à ce que l'animation soit terminée et que la barre de vie soit remplacée par une barre de vie à moitié remplie de la difficulté précédente.

Reviewer: ```suggestion
En **Stream Mode**, le joueur dispose de trois barres de vie et le joueur commence avec la deuxième barre de vie à moitié remplie au départ. Les joueurs se déplacent entre ces barres de vie grâce aux systèmes **Stream Up!** et **Stream Down!**, qui s'activent lorsqu'ils remplissent la barre de vie ou la vident complètement. Après une annonce, le jeu passe au nouveau "flux" en changeant l'arrière-plan et en passant de manière transparente à une difficulté supérieure ou inférieure de la beatmap, ainsi qu'en changeant de barre de vie. Lorsqu'un Stream Down! se produit, le joueur bénéficie d'une invincibilité temporaire pendant que la barre de vie passe à celle correspondant à la difficulté inférieure.
```

> - Default. **Lifebar will not drain at all while playing**.

Reviewer 3: ```suggestion
- Par défaut, **la barre de vie ne se vide pas tant que l'on joue**.
```

> - Default. **Lifebar will not drain at all while playing**.

Reviewer: ```suggestion
```

N'apparait pas dans la version anglaise, et je vois pas ce que ça fait la d'ailleurs

> En **Mode Expert, une barre de vie complète est présente**.

Reviewer: ```suggestion
En **Mode Expert, une seule barre de vie complète est utilisée à la place**.
```

> En termes d'obtention de vie pour la barre de vie, cela est expliqué comme ci-dessous.

Reviewer: ```suggestion
Ce qui suit remplit la barre de vie :
```

> 
> Le joueur gagne de la vie à partir de :
> 

Reviewer: ```suggestion
```

> - Hit autorisé : 300, 100, osu !, gold good ! et green good !.
>   - En touchant le(s) hit circles. (Une meilleure précision permet de gagner plus de vie. Le dernier hit circle dans une combinaison de couleurs donne plus de vie).
>   - Toucher le tick de début, de fin ou de retour du ou des sliders. (Une meilleure précision lors du tic-tac de fin permet de gagner plus de vie).
>   - Compléter correctement la ou les bulles du Spinner et du Hold.

Reviewer: ```suggestion
- Compléter les spinners et les Hold Bubbles.
- Frapper un osu!, gold good!, green good!, 300, ou un 100, 
  - Pour les cercles, les meilleurs jugements remplissent davantage la barre de vie. Le dernier cercle frappé d'une combinaison de couleurs donne plus de vie.
  - Pour les sliders, la barre de vie est remplie en fonction du nombre de parts réalisées, une fois le slider terminé.
```

> Le joueur perd de la vie :

Reviewer: ```suggestion
Ce qui suit vide la barre de vie :
```

> - Causes des coups : 50 , Miss
> - Manquer un objet ou un slider.
> - Ne pas suivre le(s) slider(s).
> - Ne pas tenir le hold circle jusqu'à ce qu'elle explose.
> - Impossible de terminer le spinner avec le message "Clear".
> - Ne pas faire tourner le spinner.

Reviewer: ```suggestion
- Recevoir un 50.
- Manquer un objet ou un slider.
- Ne pas rester dans le follow circle d'un slider.
- Ne pas tenir une Hold Bubble jusqu'à ce qu'elle explose.
- Ne pas faire tourner un spinner.
- Ne pas terminer un spinner avec le message "Clear".
```

> Le joueur garde sa barre de vie quand :
> 
> - Par défaut. **La barre de vie ne se vide pas du tout pendant le jeu.

Reviewer: ```suggestion
Contrairement aux autres versions d'osu!, la barre de vie ne se vide pas toute seule.
```

## Questions

> ### Pourquoi (insérer ici toute chanson protégée par le droit d'auteur) n'est-elle pas incluse ?

Reviewer: ```suggestion
### Pourquoi (insérer ici toute musique protégée par le droit d'auteur) n'est-elle pas incluse ?
```

> En raison de la difficulté d'obtenir une licence pour ladite chanson.

Reviewer: ```suggestion
En raison de la difficulté d'obtenir une licence pour cette dernière.
```

### Comment puis-je obtenir plus de beatmaps ?

> Rendez-vous dans la section "Store" du jeu ou appuyez sur "Télécharger plus de chansons..." dans la section "Play".

Reviewer: ```suggestion
Rendez-vous dans la section "Store" du jeu ou appuyez sur le bouton "Download more songs..." dans la section "Play".
```

### Quelle est la chanson thème de osu!stream ?

[nekodex - osu!stream theme (1.48)](https://soundcloud.com/nekodex/osu-stream-theme).
On l'entend généralement dans le menu principal et sur l'écran du tutoriel.

> ### Crédits?

Reviewer: ```suggestion
### Crédits ?
```

![](img/Os-Credit.jpg "Crédits")

> ## Links

Reviewer: ```suggestion
## Liens
```

- [Site Officiel](http://www.osustream.com) (obsolète)
- [Discussion (sur le site d'osu!)](https://osu.ppy.sh/community/forums/79)
- [Dépôt GitHub](https://github.com/ppy/osu-stream)

> - [Post de Blog](https://blog.ppy.sh/osu-stream-2020-release/)

Reviewer: ```suggestion
- [Billet de blog](https://blog.ppy.sh/osu-stream-2020-release/)
```

### Précision

`Accuracy = HitValue / (PerfectHitValue)`

| Terme | Formule |
| :-: | :-- |

> | **HitValue** | Number of misses \* 0 + Number of 50s \* 1 + Number of 100s * 2 + Number of 300s \* 4) |

Reviewer 3: Pourquoi être venu en anglais sur cette partie là ?

Reviewer: J'hésite à traduire les formules, surtout dans la première colonne vu que c'est une variable, au pire ce qui ce trouve entre parenthèses je sais pas trop non plus, j'ai pas envie d'altérer la formule en elle même.

Author: On peut laisser en français, ou en anglais. Je vois pas de problème majeur a faire de tel, donc comme vous voulez

Reviewer: Non je crois pas qu'il soit possible de simuler ça, doit bien y avoir un moyen mais je pense que c'est trop complexe, autant laisser comme ça pour le moment, au pire on modifiera en temps voulu
